### PR TITLE
Principalmapper 1.1.2 (new formula)

### PR DESCRIPTION
**Pull request by BastianZim**
Principalmapper 1.1.2 (new formula)

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/principalmapper.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    url "https://files.pythonhosted.org/packages/94/40/c396b5b212533716949a4d295f91a4c100d51ba95ea9e2d96b6b0517e5a5/urllib3-1.26.5.tar.gz"
+    sha256 "a7acd0977125325f516bda9735fa7142b909a8d01e8b2e4c8108d0984e6e0098"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    assert_equal "Account IDs:\n---", pipe_output("#{bin}/pmapper graph list").strip
```
Comment (carlocab) [inline Formula/principalmapper.rb:53]:
```suggestion
    assert_equal "Account IDs:\n---", shell_output("#{bin}/pmapper graph list").strip
```
We're not piping anything, so we don't need `pipe_output` here. Squash this change with your existing commit, please.

File: Formula/principalmapper.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
+class Principalmapper < Formula
+  include Language::Python::Virtualenv
+
+  desc "Quickly evaluate IAM permissions in AWS"
+  homepage "https://github.com/nccgroup/PMapper"
+  url "https://files.pythonhosted.org/packages/b4/4d/c270b88e05d4b17705936b7a3d05056d67e117b4087c663834d9bbd348d6/principalmapper-1.1.2.tar.gz"
+  sha256 "e36366a603b812de9eaf9c242a421f6982517b236344b9c44a1c509d82c62a26"
+  license "AGPL-3.0-only"
```
Comment (nandahkrishna) [inline Formula/principalmapper.rb:8]: This seems to indicate that it's AGPL-3.0-or-later:
https://github.com/nccgroup/PMapper/blob/6139b13e4a91cb17a5817e7127de9f3bd939eeda/principalmapper/__init__.py#L5-L8

**Comment by BastianZim** [comment]
Thank you both, has been updated.

**Review by carlocab — APPROVED** [review]

**Comment by BastianZim** [comment]
Thanks @carlocab!

**Pull request closed without merging**